sdk-python/copilotkit/copilotkit_lg_middleware.py

```
@@ -0,0 +1,183 @@
+            self,
+            state: StateSchema,
+            runtime: Runtime[Any],
+    ) -> dict[str, Any] | None:
+        # Delegate to sync implementation
+        return self.after_model(state, runtime)
+
+    # Restore frontend tool calls to AIMessage before agent exits
+    def after_agent(
+            self,
+            state: StateSchema,
+            runtime: Runtime[Any],
```

Ruff (0.14.10)

Unused method argument: `runtime`

(ARG002)